src/main/java/appeng/parts/p2p/PartP2PItems.java

```diff
@@ -49,16 +36,17 @@
 import appeng.items.parts.PartModels;
 import appeng.me.GridAccessException;
 import appeng.me.cache.helpers.TunnelCollection;
-import appeng.tile.inventory.AppEngNullInventory;
 import appeng.util.Platform;
-import appeng.util.inv.AdaptorIInventory;
-import appeng.util.inv.WrapperChainedInventory;
-import appeng.util.inv.WrapperMCISidedInventory;
+import appeng.util.inv.WrapperChainedItemHandler;
+import net.minecraft.item.ItemStack;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'net.minecraft.item.ItemStack' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)